scripts/xdebug.sh

```diff
@@ -17,6 +17,18 @@ if [ ! -f /etc/php.d/xdebug.ini ]
 then
     echo "[xdebug]
     zend_extension=\"/usr/lib64/php/modules/xdebug.so\"
+    xdebug.profiler_enable = 1
+    xdebug.profiler_enable_trigger = 1
+    xdebug.profiler_output_dir = \"/vagrant/profiling\"
     xdebug.remote_connect_back = 1
     xdebug.remote_enable = 1" > /etc/php.d/xdebug.ini
 fi
+
+if [ ! -d /vagrant/profiling ]; then
+	mkdir /vagrant/profiling
+fi
+
+#we need a bit longer to execute with profiling
+sed -i 's/max_execution_time.*/max_execution_time = 120/g' /etc/php.ini
```

Comment:
Very good point. My main thinking here is that profiling would very rarely be run. Using xdebug would literally be only for debugging, not constant use, unlike logging. 

I'm aiming to make things a lot more configurable so a sane choice here would be to make this a parameter of the script instead of hard coding. 

There was something I didn't like about outputting into the project folder, I admit.